hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/RouterMetrics.java

```diff
@@ -238,6 +255,16 @@ public long getNumSucceededAppAttemptReportRetrieved(){
     return totalSucceededAppAttemptReportRetrieved.lastStat().numSamples();
   }
 
+  @VisibleForTesting
+  public long getNumSucceededGetQueueUserAclsRetrieved(){
```

Comment:
Add tests

Reply:
Thanks for the suggestion, I will add Test.